src/ansys/speos/core/kernel/client.py

```diff
@@ -504,3 +534,29 @@ def close(self):
         self._simulationTemplateDB = None
         self._sceneDB = None
         self._jobDB = None
+        return ret_val
```

Comment:
return self.healthy as ret_val is not used anywhere else.

Reply:
retrieving self.healthy if service not tunning takes to run into timeout

Reply:
i improved the section